### DISCUSSION TITLE
SSL issue after resetting

### DISCUSSION THREAD

**Discussion by martinbenson** [nextcloud/all-in-one#4672] [Questions] [answered]
### Steps to reproduce
1. Reset the instance following: https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-properly-reset-the-instance
2. Set up a new instance

### Expected behavior 
I can connect to the new instance

### Actual behavior 
I get an SSL_ERROR_INTERNAL_ERROR_ALERT error and can't connect. The nextcloud-aio-apache log shows this (with my details obfuscated)

{"level":"error","ts":1716057427.2537725,"logger":"tls.issuance.acme.acme_client","msg":"challenge failed","identifier":"mydomain.com","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"mp.ip.address.value: Connection refused","instance":"","subproblems":[]}}
{"level":"error","ts":1716057427.2538533,"logger":"tls.issuance.acme.acme_client","msg":"validating authorization","identifier":"mydomain.com","problem":{"type":"urn:ietf:params:acme:error:connection","title":"","detail":"mp.ip.address.value: Connection refused","instance":"","subproblems":[]},"order":"https://acme-staging-v02.api.letsencrypt.org/acme/order/148664694/16615688034","attempt":1,"max_attempts":3}
{"level":"error","ts":1716057427.2539065,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"mydomain.com","issuer":"acme-v02.api.letsencrypt.org-directory","error":"HTTP 400 urn:ietf:params:acme:error:connection - mp.ip.address.value: Connection refused"}
{"level":"error","ts":1716057427.2539687,"logger":"tls.obtain","msg":"will retry","error":"[mydomain.com] Obtain: [mydomain.com] solving challenge: mydomain.com: [mydomain.com] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - mp.ip.address.value: Connection refused (ca=https://acme-staging-v02.api.letsencrypt.org/directory)","attempt":5,"retrying_in":600,"elapsed":606.521083906,"max_duration":2592000}

### Host OS 
Ubuntu 22.04

#### Nextcloud AIO version 
8.2.1

#### Current channel 
latest

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/2105

**Comment by martinbenson** [comment]
Thanks for getting back! I saw that issue but I don't think the conditions there apply. I don't use cloudflare and there are no AAAA entries in DNS. There are some CNAME entries of course but none of them relate to nextcloud as far as I see - they're related to other subdomains. Are there any other diagnostic steps I can take to get to the root cause? I guess I could change the name of the nextcloud subdomain - wouldn't be a massive issue, but if there's a different solution that would be better!
